src/components/Header.js

```diff
@@ -11,28 +11,28 @@ const Header = () => {
                 <NavItem>
                     <NavLink
                         active
-                        href="#"
+                        href="./Home"
                     >
                         Home
                     </NavLink>
                 </NavItem>
                 <NavItem>
-                    <NavLink href="#">
+                    <NavLink href="/SVIndex">
                         Index
                     </NavLink>
                 </NavItem>
                 <NavItem>
                     <NavLink
-                        disabled
-                        href="#"
+                       
+                        href="/SVNew"
```

Comment:
make this inline with it's tage. It's minor but looks better to employers who review your code.